code/core/src/manager-api/modules/versions.ts

```diff
@@ -50,7 +50,12 @@ export interface SubAPI {
    *
    * @returns {string} The URL of the Storybook Manager documentation.
    */
-  getDocsUrl: (options: { subpath?: string; versioned?: boolean; renderer?: boolean }) => string;
+  getDocsUrl: (options: {
+    asset?: boolean;
```

Comment:
It's not obvious what this new `asset` flag is for, or what it does.
Can you add a JSdoc comment?